server/util/outputNormalization.test.js

```diff
@@ -80,7 +127,36 @@ describe('outputsMatch', () => {
   it('returns true for strings that normalize to the same value', () => {
     expect(outputsMatch('  hello   world  ', 'hello world')).toBe(true);
     expect(outputsMatch('hello\nworld', 'hello world')).toBe(true);
-    expect(outputsMatch('HELLO\t\tWORLD', 'HELLO WORLD')).toBe(true);
+    expect(outputsMatch('Hello\t\tWorld', 'hello world')).toBe(true);
+  });
+
+  it('returns true for strings with different punctuation', () => {
+    expect(outputsMatch('Hello, World!', 'Hello World')).toBe(true);
+    expect(outputsMatch('Button "Save"', 'Button Save')).toBe(true);
+    expect(outputsMatch('Press Enter.', 'Press Enter')).toBe(true);
+  });
+
+  it('returns true for strings with different hyphen spacing', () => {
+    expect(
+      outputsMatch('Control-Option-Space', 'Control - Option - Space')
+    ).toBe(true);
+    expect(
+      outputsMatch('Control  -  Option  -  Space', 'Control-Option-Space')
+    ).toBe(true);
+  });
+
+  it('returns true for strings with different capitalization patterns', () => {
+    expect(outputsMatch('Hello World', 'hello world')).toBe(true);
+    expect(outputsMatch('HeLLo WoRLd', 'heLLo woRLd')).toBe(true);
+  });
+
+  it('returns true for strings with JAWS separator tokens', () => {
+    expect(
+      outputsMatch('Text\u001dSeparator\u001eMore', 'Text Separator More')
+    ).toBe(true);
+    expect(outputsMatch('Start\u001dMiddle\u001eEnd', 'Start Middle End')).toBe(
+      true
+    );
```

Comment:
Could we add one test case for `"Start \u001d Middle \u001e End"`? That string more closely resembles the behavior we've observed, and the test case will prevent future revisions from collapsing white space before making the substitution.

Reply:
Good call! [6b85938](https://github.com/w3c/aria-at-app/pull/1459/commits/6b859389810eb3b28800376eec78afdacf99c4ea)